NEWS.md

```diff
@@ -1,3 +1,14 @@
+# New in snapd 2.61.1:
+* Added prerequisite tracker for self-contained snap sets
+* Fix auth.json access for login as non-root group ID
```

Comment:
Please double check my understanding here.

Reply:
This looks correct to me, but perhaps @pedronis can chip in